config/config_test.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"os"
+	"testing"
+)
+
+func TestLoadConfig(t *testing.T) {
```

Comment:
I think that it would be good to have at least one test for the apikey type as well. Technically you could set both configs, but the type should determine what is used in the code.

Reply:
added test for apikey